conf/mesos-site.xml

```diff
@@ -34,22 +34,24 @@
 
```

Comment:
Please don't change these config files. They are general settings used on a typical hdfs cluster, which will typically allocate adequate space for each node. They were also modeled after airbnb's settings.